service-base/src/main/java/org/eclipse/hono/service/credentials/BaseCredentialsService.java

```diff
@@ -110,28 +110,32 @@ protected void doStop(final Future<Void> stopFuture) {
     public final void processCredentialsMessage(final Message<JsonObject> regMsg) {
 
         final JsonObject body = regMsg.body();
-        if (checkMessagePartsForNull(body,"body",regMsg)) {
+        if (body == null) {
+            log.info("credentials request did not contain body - not supported");
```

Comment:
this should not be logged at INFO level but at DEBUG. We do expect this to happen (frequently) since clients can basically send what they want to. However, this doesn't pose any threat to Hono so we should log this at DEBUG in case we want to find out why a particular client cannot access the API ...

Same holds true for the other occurrences below ...